translations/TablerBundle.fr.xliff

```diff
@@ -106,6 +106,26 @@
                 <source>Take me home</source>
                 <target>Retourner au début</target>
             </trans-unit>
+            <trans-unit id="Password Reset Email Sent">
+                <source>Password Reset Email Sent</source>
+                <target>Email réinitialisation de mot de passe envoyé</target>
```

Comment:
This English doesn't make sense to me. Seems to be a 1:1 translations from Google translate. Can you explain what that Eans?

Reply:
Doesn't come from me. 
This is the label created when using `make:reset-password`

Reply:
We should stick to `symfonycasts/reset-password-bundle` for variable names.
But for trans, we are free.

What do you want to put instead ?